### ISSUE TITLE
Blowing Hodir's Horn Daily Quest (quest=12977) Completion text incorrect

### ISSUE THREAD

**Issue by Xandriarn** [dalaranwow/dalaran-wow#4632]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Upon completing the quest, http://www.wowhead.com/quest=12977/blowing-hodirs-horn, the turn-in text tells you to return to King Jokkum when you really turn the quest back in at the horn.

**Expected behaviour**: Quest completion text should be updated to reflect proper turn-in location of Hodir's Horn (http://www.wowhead.com/object=192078/hodirs-horn).

**Steps to reproduce the problem**:

1. Accept daily quest Blowing Hodir's Horn
2. Complete Quest
3. Observe Competion text

**Include proofs for this behaviour**

Completed quest while looking at King Jokkum

![wowscrnshot_032417_132515](https://cloud.githubusercontent.com/assets/25491266/24308665/ef874072-1096-11e7-8a8b-d24f7d5b1413.jpg)

Completed quest while looking at Hodir's Horn

![wowscrnshot_032417_132500](https://cloud.githubusercontent.com/assets/25491266/24308666/ef8b0374-1096-11e7-8567-091b895141ff.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
-Those texts are from wotlk (2008-2010) and they are all correct. So if the description is wrong it was wrong back in the wotlk time too.

It's a blizzlike issue it won't be changed. Thank you anyway for the report.

Source from sniff from wotlk time (07/17/2010 02:13:16.000)

```sql
ServerToClient: SMSG_QUERY_QUEST_INFO_RESPONSE (0x005D) Length: 1179 ConnIdx: 0 Time: 07/17/2010 02:13:16.000 Number: 168256
Quest ID: 12977
QuestType: Normal (2)
QuestLevel: 80
QuestMinLevel: 77
QuestSortID: 67 (67)
QuestInfoID: None (0)
SuggestedGroupNum: 0
[0] RequiredFactionID: 0
[0] RequiredFactionValue: 0
[1] RequiredFactionID: 0
[1] RequiredFactionValue: 0
NextQuestID: 0 (0)
RewardXPDifficulty: 5
RewardMoney: 74000
RewardBonusMoney: 132300
RewardDisplaySpell: 0 (0)
RewardSpell: 0 (0)
RewardHonor: 0
RewardKillHonor: 0
StartItem: Hodir's Horn (42164)
Flags: ExpansionOnly, Daily (4224)
RewardTitle: 0
RequiredPlayerKills: 0
RewardTalents: 0
RewardArenaPoints: 0
Unknown Int32: 0
[0] RewardItems: 0 (0)
[0] RewardAmount: 0
[1] RewardItems: 0 (0)
[1] RewardAmount: 0
[2] RewardItems: 0 (0)
[2] RewardAmount: 0
[3] RewardItems: 0 (0)
[3] RewardAmount: 0
[0] RewardChoiceItemID: 0 (0)
[0] RewardChoiceItemQuantity: 0
[1] RewardChoiceItemID: 0 (0)
[1] RewardChoiceItemQuantity: 0
[2] RewardChoiceItemID: 0 (0)
[2] RewardChoiceItemQuantity: 0
[3] RewardChoiceItemID: 0 (0)
[3] RewardChoiceItemQuantity: 0
[4] RewardChoiceItemID: 0 (0)
[4] RewardChoiceItemQuantity: 0
[5] RewardChoiceItemID: 0 (0)
[5] RewardChoiceItemQuantity: 0
[0] RewardFactionID: 1119
[1] RewardFactionID: 0
[2] RewardFactionID: 0
[3] RewardFactionID: 0
[4] RewardFactionID: 0
[0] RewardFactionValue: 6
[1] RewardFactionValue: 0
[2] RewardFactionValue: 0
[3] RewardFactionValue: 0
[4] RewardFactionValue: 0
[0] RewardFactionOverride: 0
[1] RewardFactionOverride: 0
[2] RewardFactionOverride: 0
[3] RewardFactionOverride: 0
[4] RewardFactionOverride: 0
POIContinent: 0
POIx: 0
POIy: 0
POIPriority: 0
LogTitle: Blowing Hodir's Horn
LogDescription: You are to defeat 5 Niffelem Forefathers and 5 Restless Frostborn at Thunderfall. You are then to blow Hodir's Horn over their corpses, then return to Hodir's Horn in Dun Niffelem.
QuestDescription: You have done a great service to the giants of Dun Niffelem. Without your aid, this monument would not exist.$b$bAs you silently ponder its significance, feelings begin to flood your heart. The feelings fill your mind with thoughts and images... as well as a sense of urgency.$b$bThe sensations fade and leave you with a knowledge of what you must do. The restless spirits at Thunderfall must be put to rest.$b$bAs you set off, you find that your hands have been filled with an ethereal horn.
AreaDescription: 
QuestCompletionLog: Return to King Jokkum at Dun Niffelem in The Storm Peaks.
[0] Required: NPC ID: 30138 (Frost Giant Ghost KC)
[0] RequiredCount: 5
[0] RequiredItemID: 0 (0)
[0] RequiredItemCount: 0
[1] Required: NPC ID: 30139 (Frost Dwarf Ghost KC)
[1] RequiredCount: 5
[1] RequiredItemID: 0 (0)
[1] RequiredItemCount: 0
[2] Required: NPC ID: 0
[2] RequiredCount: 0
[2] RequiredItemID: Hodir's Horn (42164)
[2] RequiredItemCount: 1
[3] Required: NPC ID: 0
[3] RequiredCount: 0
[3] RequiredItemID: 0 (0)
[3] RequiredItemCount: 0
[0] RequiredItemID: 0 (0)
[0] RequiredItemCount: 0
[1] RequiredItemID: 0 (0)
[1] RequiredItemCount: 0
[2] RequiredItemID: 0 (0)
[2] RequiredItemCount: 0
[3] RequiredItemID: 0 (0)
[3] RequiredItemCount: 0
[4] RequiredItemID: 0 (0)
[4] RequiredItemCount: 0
[5] RequiredItemID: 0 (0)
[5] RequiredItemCount: 0
[0] Objective Text: Niffelem Forefather freed
[1] Objective Text: Restless Frostborn freed
[2] Objective Text: 
[3] Objective Text: 
```
